doc/include/api_introspection.inc

```diff
@@ -1,144 +1,95 @@
-as its own Tcl companion file (see [cmd critcl::tsources]) to
-distinguish between sourced by mode "compile & run" for compilation
-and sourced from either the result of mode "generate package" or during
-the load phase of "compile & run".
+[para] Enables a [vset critcl_script] used as its own Tcl companion file (see
+[cmd critcl::tsources]) to distinguish between being sourced for compilation in
+[term {compile & run}] mode and being sourced from either the result of
+[term {generate package}] mode or during the load phase of
+[term {compile & run}] mode.
 
-In case of the two latter possibilities the result is [const true],
-and [const false] for the first.
+The result is [const false] in the first case and [const true] in the later two
+cases.
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::compiling]]
 
-This command returns a boolean value. It returns [const true] if C
-code can be compiled on this platform in general, i.e. if a C compiler
-is available, and [const false] otherwise.
+Returns [const true] if a working C compiler is available and [const false]
+otherwise.
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::done]]
 
-This command returns a boolean value. It returns [const true] when
-critcl has built the embedded C code, and [const false] otherwise.
+Returns [const true] when [vset critcl_script] has been built and
+[const false] otherwise.  Only useful from within a [vset critcl_script].
+Enables the Tcl parts of a [vset critcl_script] to distinguish between
+[term {prebuilt package}] mode and [term {compile & run}] mode.
 
-[para] This enables the Tcl code of a critcl-based package to
-distinguish between it getting used as a prebuilt package, versus
-dynamic compile & run, and take action based on that.
-
-[para] [emph Note] that this command is only useful from within a
-[file .critcl] file. The result is managed on a per-file basis, like
-is done for the commands embedding C code and controlling the
-behaviour of compiler and linker.
-
-[para] See also section [sectref {Modes Of Operation/Use}].
+[para] See also [sectref {Modes Of Operation/Use}].
 
 [comment ---------------------------------------------------------------------]
 [call [cmd ::critcl::failed]]
 
-This command returns a boolean value. It returns [const true] if
-critcl has failed to build the package, and [const false] otherwise,
-i.e. success. As part of this it forces the building of the package,
-but not its loading. Note that it will attempt to build the package
-only on the first call; future calls for the same package will return
-a cached result.
-
-[para] This enables a critcl-based package to check itself for
-availability and throw an error if it could not be built. [emph Note]
-that the command does [emph not] throw such an error itself.
+Returns [const true] the [vset critcl_script] could not be built, and
```

Comment:
`the` -> `if the`

Reply:
Done with commit 29380521d0873ea1dcc5f8cdff6742b20764473f (Branch `documentation` in my repo, not seen by the PR).